src/gui/symbology/qgsinterpolatedlinesymbollayerwidget.h

```diff
@@ -0,0 +1,80 @@
+/***************************************************************************
+  qgsinterpolatedlinesymbollayerwidget.h - QgsInterpolatedLineSymbolLayerWidget
+
+ ---------------------
+ begin                : 23.3.2021
+ copyright            : (C) 2021 by Vincent Cloarec
+ email                : vcloarec at gmail dot com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+#ifndef QGSINTERPOLATEDLINESYMBOLLAYERWIDGET_H
+#define QGSINTERPOLATEDLINESYMBOLLAYERWIDGET_H
+
+#include "qgsrendererwidget.h"
+#include "qgssymbollayerwidget.h"
+#include "qgsinterpolatedlinerenderer.h"
+#include "ui_qgsinterpolatedlinesymbollayerwidgetbase.h"
+
+#ifndef SIP_RUN
```

Comment:
Is this intentional? That gives us empty `qgsinterpolatedlinesymbollayerwidget.sip.in` file... for consistency with other symbol layer widgets it would be good to include it in SIP.